dodo.py

```diff
@@ -819,7 +820,7 @@ class C:
     TESTING_IN_CI = json.loads(os.environ.get("TESTING_IN_CI", "0"))
     WIN_DEV_IN_CI = json.loads(os.environ.get("WIN_DEV_IN_CI", "0"))
     PYM = [sys.executable, "-m"]
-    FLIT = [*PYM, "flit"]
+    HATCH = [*PYM, "hatch"]
```

Comment:
is there special `hatch` things, or can we go to `pyproject-build`?

Reply:
No `hatch` specific things. For now it's really just a matter of aligning with the other Jupyter projects as mentioned in the top comment:

> Fixes https://github.com/jupyterlite/jupyterlite/issues/881 for consistency with other packages and https://blog.jupyter.org/packaging-for-jupyter-in-2022-c7be64c38926.

So at least the configuration in `pyproject.toml` looks similar as for the other projects and one does not need to learn about a new tool. I guess we could consider `pyproject-build` if that simplifies even further, something to track in another issue?